src/vaadin-dialog-resizable-mixin.html

```diff
@@ -0,0 +1,179 @@
+<dom-module id="vaadin-dialog-resizable-overlay-styles" theme-for="vaadin-dialog-overlay">
+  <template>
+    <style>
+      [part='overlay'] {
+        position: relative;
+        overflow: visible;
+      }
+
+      .resizer-container {
+        height: 100%;
+        overflow: auto;
+      }
+
+      :host(:not([resizable])) .resizer {
+        display: none;
+      }
+
+      .resizer {
+        position: absolute;
+        height: 10px;
+        width: 10px;
+      }
+
+      .resizer.edge {
+        top: -5px;
+        right: -5px;
+        bottom: -5px;
+        left: -5px;
+      }
+
+      .resizer.edge.n {
+        width: auto;
+        bottom: auto;
+        cursor: n-resize;
```

Comment:
All the resize cursors should be those that have an arrow pointing both directions, `ns-resize`, `ew-resize`, `nesw-resize` and `nwse-resize`.

Reply:
I've changed them now